CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 
 ## Unreleased
 
+### New features
+
+- New options `useEssSession` for `session.handleIncomingRedirect`: If set to false, 
```

Comment:
```suggestion
- New option `useEssSession` for `session.handleIncomingRedirect`: If set to false, 
```

And maybe a short mention of how this behaviour was introduced in 1.4.0 so people can reference back to that would be helpful too :)

Reply:
Thanks for the suggestions :) 6b9de5b95196d8ba9c55ffa51e18da875248887f